Gopkg.toml

```diff
@@ -6,6 +6,18 @@
   name = "github.com/aws/aws-sdk-go"
   version = "v1.10.8"
 
+[[constraint]]
```

Comment:
Are these new constraints necessary? Or were they just needed for when you were pointing to a branch?

Reply:
I needed them to get dep to successfully find a solution, but now it has, I don't seem to need them.  Removed.

Reply:
Looks like weaveworks/common is still new and unneeded